cmake/modules/SetUpWindows.cmake

```
@@ -29,15 +29,15 @@ elseif(MSVC)
   math(EXPR VC_MINOR "${MSVC_VERSION} % 100")
 
   if(winrtdebug)
-    set(BLDCXXFLAGS "-Zc:__cplusplus -MDd -GR")
+    set(BLDCXXFLAGS "-Zc:__cplusplus -std:c++${CMAKE_CXX_STANDARD} -MDd -GR")
```

```suggestion
    set(BLDCXXFLAGS "-std:c++${CMAKE_CXX_STANDARD} -MDd -GR")
```
`__cplusplus` doesn't seem to be needed anymore these days? https://www.godbolt.org/z/Kjfb1Tfa5

---

It is, otherwise it is always 1997:  `int i = 199711L;`

---

Argh sorry misunderstood!